CalibCalorimetry/HcalTPGAlgos/src/LutXml.cc

```diff
@@ -296,37 +298,27 @@ int LutXml::test_access(std::string filename) {
   return 0;
 }
 
-HcalSubdetector LutXml::subdet_from_crate(int crate_, int eta, int depth) {
-  HcalSubdetector result;
-  // HBHE: 0,1,4,5,10,11,14,15,17
-  // HF: 2,9,12
-  // HO: 3,6,7,13
+HcalSubdetector LutXml::subdet_from_crate(int crate_, int slot, int fiber) {
+  // HBHE: 0,1,4,5,10,11,14,15,17 (+20)
+  // HF: 2,9,12  (+20)
+  // HO: 3,6,7,13,18  (+20)
   int crate = crate_ < 20 ? crate_ : crate_ - 20;
-
   if (crate == 2 || crate == 9 || crate == 12)
-    result = HcalForward;
-  else if (crate == 3 || crate == 6 || crate == 7 || crate == 13)
-    result = HcalOuter;
+    return HcalForward;
+  else if (crate == 3 || crate == 6 || crate == 7 || crate == 13 || crate == 18)
```

Comment:
What is crate 18? It's not listed in the electronics map.

Reply:
it's a utca crate 38